src/web/mjs/connectors/BentoManga.mjs

```diff
@@ -0,0 +1,79 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class bentomanga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'bentomanga';
+        super.label = 'Bentô Manga';
+        this.tags = [ 'manga', 'webtoon', 'novel', 'french' ];
+        this.url = 'https://bentomanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1'); 
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim(); 
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(this.url + '/manga_list', this.requestOptions); 
+        let data = await this.fetchDOM(request, 'div.div-manga_cover a'); 
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+    }    
+
+    async _getChapters(manga) {
+        let request = new Request(this.url + manga.id, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.component-chapter-title > a');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.querySelector('span.chapter_volume').innerText.trim() + ' ' + element.querySelector('span.chapter_title').innerText.trim(),
+                language: ''
+            };
+        });
+    }
+
+    async _getPages(chapter) {
+        const script = `
+            new Promise(async (resolve, reject) => {
+                if(document.querySelector('form#captcha-form')) {
+                    return reject(new Error('The chapter is protected by reCaptcha! Use the manual website interaction to solve the Captcha for an arbitrary chapter before downloading any other chapter from this website.'));
+                }
+                const info = document.querySelector('head meta[data-chapter-id]');
+                const uri = new URL('/api/?type=chapter&id=' + info.dataset.chapterId, window.location.origin);
+                const customHeaders = {
+                    headers: {"a":"Math.random().toString(16)"}
```

Comment:
Apparently, no need for "a" in custom headers. But you will need ('X-Requested-With', 'XMLHttpRequest') like before; 

``` headers : { 'X-Requested-With' : 'XMLHttpRequest' } ```